app/Providers/MediawikiServiceProvider.php

```diff
@@ -36,6 +36,7 @@ public function register()
             try {
                 Log::debug('Connect to Mediawiki');
                 $api = new MediawikiApi(env('WIKI_URL').'/api.php');
+                Log::debug('Log in');
```

Comment:
Happy for this change to go in, though seems unrelated to this particular issue.